Adding a new PHP array (#3149)
Codecademy/docs: content/php/concepts/arrays/terms/array-values/array-values.md

---
Title: 'array_values()'
Description: 'All of the values in the array are returned.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Arrays'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

> The **`array_values()`** function allows you to return all the values from an array in numerical order.

Reviewer: ```suggestion
The **`array_values()`** function returns all the values from an array in numerical order.
```

## Syntax

> ```

Reviewer: ````suggestion
```pseudo
````

array_values(array)
```

> **Note:** The array that is returned will have numeric keys that start from 0 and is incremented by 1.

> ## Example
> The example shown below creates an array called `$vehicle` with three key-value pairs. It then uses the `array_values()` function to extract and print the array's values resulting in an indexed array.
> ```

Reviewer: ````suggestion
## Example

The example shown below creates an array called `$vehicle` with three key-value pairs. It then uses the `array_values()` function to extract and print the array's values resulting in an indexed array.

```php
````

<?php
$vehicle = array("car" => "BMW", "color" => "gold", "model" => "X5");
print_r(array_values($vehicle));
?>

> ```
> This show the values "BMW", "gold" and "X5" that are printed to the console.
> ```shell

Reviewer: ````suggestion
```

This example shows the values "BMW", "gold" and "X5" printed to the console:

```shell
````
